src/library/parsercsv.cpp

```diff
@@ -129,7 +133,7 @@ bool ParserCsv::writeCSVFile(const QString &file_str, BaseSqlTableModel* pPlayli
     if (!file.open(QIODevice::WriteOnly | QIODevice::Text)) {
         QMessageBox::warning(nullptr,
                 tr("Playlist Export Failed"),
-                tr("Could not create file") + " " + file_str);
+                tr("Could not create file") + " " + file_str + "\n" + file.errorString());
```

Comment:
This is possibly not localized and may look weird to inexperienced users. I think we should use the property-based API here and use the [detailed text](https://doc.qt.io/qt-5/qmessagebox.html#detailedText-prop) for this.